content/news/playtest-20150102.md

```diff
@@ -0,0 +1,34 @@
+We wish everyone a happy new year and prepared a new playtest which builds upon our [Halloween release](/news/release-20141029/). The main focus was to improve stability and performance as well as implement unique features of Dune 2000 that distinguish it from the other mods.
```

Comment:
"The main focuses of this release were improving stability and performance, as well as implementing some of the missing features that make Dune 2000 unique." reads a bit better IMO.